modules/ROOT/pages/tgw-troubleshoot.adoc

```diff
@@ -0,0 +1,79 @@
+= Troubleshoot Transit Gateway Attachments
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+image:logo-cloud-active.png[link="/runtime-manager/deployment-strategies#cloudhub", title="CloudHub"]
+image:logo-hybrid-disabled.png[link="/runtime-manager/deployment-strategies#hybrid", title="Hybrid"]
+image:logo-server-disabled.png[link="/runtime-manager/deployment-strategies#anypoint-platform-private-cloud-edition", title="Anypoint Platform PCE"]
+image:logo-rtf-disabled.png[link="/runtime-manager/deployment-strategies#anypoint-runtime-fabric", title="Runtime Fabric"]
+
+These are some of the possible errors you might encounter when attempting to attach to a transit gateway.
+
+== Unable to Attach to a Transit Gateway
+
+If you see the `Resource Share Not Found` error when attempting to attach to the 
+transit gateway:
+
+. Verify that the *ID* and *Owner* values that you specified are correct. 
+. Click the pencil icon to edit these values.
+
+Alternatively, remove the transit gateway and re-add it.
+See xref:tgw-manage-arm.adoc#remove-a-transit-gateway-from-anypoint-platform[Remove a Transit Gateway from Anypoint Platform] and xref:tgw-add-arm.adoc#attach-a-vpc-to-the-transit-gateway[Attach a VPC to the Transit Gateway].
+
+
+== Unable to Delete a Transit Gateway
+
+If the *Transit Gateways* page doesn't list any VPCs, but you can't delete the transit gateway, the transit gateway might be attached to a VPC that belongs to another business group that you don't have access to.
+
+== Unable to View a Transit Gateway
+
+Runtime Manager always adds transit gateways at the root business group level.
+To view a transit gateway, you need CloudHub Network Viewer permission.
+
+== Unable to Modify a Transit Gateway
+
+To modify a transit gateway, you need CloudHub Network Admin permission.
+
+
+== Transit Gateway Status is Unknown
+
+If the status for the transit gateway is `Unknown`, click the *View on AWS* button on the *Transit Gateways* page to resolve the issue on AWS:
```

Comment:
In this case, we can infer that the TGW status is unknown because the resource share has been disabled or deleted. If it's only been disabled, they can enable it to address the issue. Up to you whether that's worth adding here or not.